vnext/Shared/OInstance.cpp

```diff
@@ -235,12 +235,14 @@ struct BridgeUIBatchInstanceCallback : public InstanceCallback {
         if (uiManager != nullptr)
           uiManager->onBatchComplete();
       });
+#ifdef WINRT
```

Comment:
Please add a comment that WIN32 can't use dynamic_cast (is this due to not having RTTI turned on?)

Reply:
Yes, Office code base doesn't have RTTI.

Reply:
I went ahead and added some comments here.